ui/components/User.js

```diff
@@ -48,72 +45,64 @@ function exportToJsonFile(jsonData, filename) {
   linkElement.remove()
 }
 
-class User extends React.Component {
-  state = {
-    user : null,
-    open : false,
-    account : ExtensionPointSchemaValidator("account")(),
-    providerType : '',
-    capabilitiesLoaded : false
-  }
+function User(props){
+  const [user, setUser] = useState(null);
+  const [open, setOpen] = useState(false);
+  const [account, setAccount] = useState([]);
+  const [capabilitiesLoaded, setCapabilitiesLoaded] = useState(false);
+  const anchorEl = useRef(null);
+  const dispatch = useDispatch();
+
+  const capabilitiesRegistry = useSelector(state => state.get("capabilitiesRegistry"));
 
-  handleToggle = () => {
-    this.setState((state) => ({ open : !state.open }));
+  const handleToggle = () => {
+    setOpen(!open);
   };
 
-  handleClose = (event) => {
-    if (this.anchorEl.contains(event.target)) {
+  const handleClose = (event) => {
+    if (anchorEl.current && anchorEl.current.contains(event.target)) {
       return;
     }
-    this.setState({ open : false });
+    setOpen(false);
   }
 
-  handleLogout = () => {
+  const handleLogout = () => {
     window.location = '/user/logout'
   };
 
-  handleError = (error) => {
-    const notify = this.props.notify;
```

Comment:
We need to use the `useNotification` hook here.

Reply:
Since we have removed the `handleError` function it is not being used anywhere. 
cc. @Yashsharma1911 ( [ref](https://github.com/meshery/meshery/pull/8834#discussion_r1335043232) )

So shall I remove the `userNotification` altogether?